src/main/java/com/maxmind/db/Reader.java

```diff
@@ -149,6 +153,10 @@ public <T> T get(InetAddress ipAddress, Class<T> cls) throws IOException {
         return getRecord(ipAddress, cls).getData();
     }
 
+    protected int getIpv4Start() {
```

Comment:
I think it would make more sense if this and the other methods were package private rather than protected. I suppose it doesn't matter too much as the class is final, but I think it would make it clearer that it isn't part of the API.